src/main/kotlin/no/nav/amt/distribusjon/varsel/VarselJob.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.amt.distribusjon.varsel
+
+import io.ktor.util.Attributes
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import no.nav.amt.distribusjon.application.isReadyKey
+import no.nav.amt.distribusjon.journalforing.job.leaderelection.LeaderElection
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.time.Duration
+import java.time.temporal.ChronoUnit
+import java.util.Timer
+import kotlin.concurrent.fixedRateTimer
+
+class VarselJob(
+    private val leaderElection: LeaderElection,
+    private val attributes: Attributes,
+    private val varselService: VarselService,
+) {
+    private val log: Logger = LoggerFactory.getLogger(javaClass)
+    private val scope = CoroutineScope(Dispatchers.IO)
+
+    fun startJob(): Timer {
+        return fixedRateTimer(
+            name = this.javaClass.simpleName,
+            initialDelay = Duration.of(5, ChronoUnit.MINUTES).toMillis(),
```

Comment:
Hadde det vært lurt å justere disse med et minutt eller to så den ikke kjører nøyaktig samtidig som den andre jobben?

Reply:
Det kan vi. Jeg vet ikke hvor ofte jobben bør kjøres egentlig.